packages/player/package.json

```diff
@@ -75,7 +75,7 @@
     "shaka-player": "^4.7.4",
     "sinon": "^17.0.0",
     "typescript": "^5.2.2",
```

Comment:
@enjikaka I think it looks like the usage of "hats" in the version ranges in Player makes Renovate confused? Should we do like in the other packages and use exact versions?

The problem being this PR not updating all patch/minor release as it should :-)

Reply:
Sure we can remove the hats. This sounds like a bug in Renovate? `typescript@^5.2.2` allows  `5.2.2`, `5.3.2` and `5.3.3` according to the tool at https://semver.npmjs.com/. 🤔

Reply:
Yes it looks like a bug on their side, but it might be cleaner if do it the same in all packages anyway :-) 

If there are packages you'd like to block it from updating we can list those in: https://github.com/tidal-music/tidal-sdk-web/blob/main/renovate.json

Reply:
I did a fix in: #44 :-)